src/libraries/Microsoft.PowerFx.Core/Parser/ParseFormulasResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Syntax.Nodes;
+using Microsoft.PowerFx.Core.Utils;
+using System.Collections.Generic;
+
+namespace Microsoft.PowerFx.Core.Parser
+{
+    internal class ParseFormulasResult
```

Comment:
>ParseFormulasResult [](http://example.com/codeflow?start=19&length=19)

nit: can you match the formatting / styel in the ParseResult class?

Reply:
I believe the styling matches now, let me know if I'm missing something.